### ISSUE TITLE
Proposals from unknown Peer

### ISSUE THREAD

**Issue by ggwpez** [perun-network/perun-eth-payment#15]
**Problem**: A `ChannelProposal` currently contains a `From Peer` field. This works fine as long as we only accept proposals from peers which were previously registered with `RegisterPeer`.  
Now the problem is that the payment echo bot wants to accept channel proposals from addresses which where not registered.  
The current logic is to reject these proposals, since returning a `Peer` which was not created by the user with `RegisterPeer` does not make sense.

**Possible solutions**:
1. Remove `Peer` and replace with `common.Address`, similar to #11. We could also remove `RegisterPeer` in this step.
2. Add a `FromAddress common.Address` and rename `From Peer` to `FromPeer *Peer` in the channel proposal.
    Only one if the two field would be set then, depending on whether or not the peer is known.
3. Change `From Peer` to `From common.Address`

**Comment by matthiasgeihs** [comment]
Change `From Peer` to `From common.Address`?

**Comment by ggwpez** [comment]
> Change `From Peer` to `From common.Address`?

Yes we could also do that. I guess that would be the fastest fix for now.
